Kernel/Devices/Storage/USB/SCSIComands.h

```diff
@@ -13,24 +13,145 @@
 // https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf
 
 namespace Kernel::SCSI {
-// 3.22.1
-struct ReadCapacity10 {
-    u8 opcode { 0x25 };
-    u8 reserved1 { 0 };
-    BigEndian<u32> oboslete_logical_block_address { 0 };
-    u16 reserved2 { 0 };
-    u8 reserved3 { 0 };
+
+// 2.4.1.2
+struct FixedFormatSenseData {
+    u8 response_code : 7;
+    u8 valid : 1;
+    u8 obsolete;
+    u8 sense_key : 4;
+    u8 reserved : 1;
+    u8 ili : 1;
+    u8 eom : 1;
+    u8 file_mark : 1;
+    BigEndian<u32> information;
+    u8 additional_sense_length;
+    BigEndian<u32> command_specific_information;
+    u8 additional_sense_code;
+    u8 additional_sense_code_qualifier;
+    u8 field_replaceable_unit_code;
+    u8 sense_key_specific[3];
+    u8 additional_sense_bytes[];
+};
+static_assert(AssertSize<FixedFormatSenseData, 18>());
+
+// 3.6.1
+struct Inquiry {
+    u8 opcode { 0x12 };
+    u8 enable_vital_product_data { 0 }; // EVPD, Also CMDDT in the second bit, but thats obsolete
+    u8 page_code { 0 };
+    BigEndian<u16> allocation_length;
     u8 control { 0 };
 };
-static_assert(AssertSize<ReadCapacity10, 10>());
-// 3.22.2
-struct ReadCapacity10Parameters {
-    BigEndian<u32> block_count;
-    BigEndian<u32> block_size;
+static_assert(AssertSize<Inquiry, 6>());
+
+// 3.6.2
+struct StandardInquiryData {
+    // Table 61
+    enum class DeviceType : u8 {
+        DirectAccessBlockDevice = 0x00,
+        SequentialAccessDevice = 0x01,
+        PrinterDevice = 0x02,
+        ProcessorDevice = 0x03,
+        WriteOnceDevice = 0x04,
+        CDDVDDevice = 0x05,
+        // 0x06 was Scanner device
+        OpticalMemoryDevice = 0x07,
+        MediumChangerDevice = 0x08,
+        // 0x09 was Communications device
+        // 0x0A-0x0B are obsolete
+        StorageArrayControllerDevice = 0x0C,
+        EnclosureServicesDevice = 0x0D,
+        SimplifiedDirectAccessDevice = 0x0E,
+        OpticalCardReaderWriterDevice = 0x0F,
+        BridgeControllerCommands = 0x10,
+        ObjectBasedStorageDevice = 0x11,
+        AutomationDriveInterface = 0x12,
+        // 0x13-0x1D are reserved
+        WellKnownLogicalUnit = 0x1E,
+        UnknownOrNoDeviceType = 0x1F,
+    };
+    struct {
+        DeviceType device_type : 5;
+        u8 qualifier : 3;
+    } peripheral_info;
+    u8 removable; // 0x80 for removable, 0x00 for fixed
+    u8 version;
+    struct {
+        u8 response_data_format : 4;
+        u8 hierarchical_support : 1; // HISUP
+        u8 normal_aca_support : 1;   // NORMACA
+        u8 obsolete : 2;
+    } response_data;
+    u8 additional_length; // N-4
+    struct {
+        u8 protect : 1;
+        u8 reserved : 2;
```

Comment:
This should probably also just be `u8 : 2;` just like the other reserved fields.